jest.config.js

```diff
@@ -3,6 +3,8 @@
 const coverageIsWanted = !!process.env.RXP_COVERAGE;
 
 module.exports = {
+  // to uncomment to display logs.
+  // verbose: false,
```

Comment:
Note: It sounds like it's the other way around (that verbose should be set to true) but it's not.
I don't get it but well, it's how it works.